interface/forms/eye_mag/help.php

```diff
@@ -35,411 +34,572 @@
-                        <br />
-                        Look around the form - openEMR: Eye Exam is automatically filled.<br />
-                        Done. No extra clicks.<br />
-
-                        <hr />
-                        This tutorial shows you how to document each area using Shorthand.  <br />
-                        We'll show you how to complete the HPI, PMH, POH, Medication list, Surgical History and Allergies.<br />
-                        As an example, using a handful of lines of typing in the Shorthand textbox,<br />
-                        you will document all your normal findings <strong>and more than 40 different clinical issues</strong>.
+    <body>
+        <!-- Navbar Section -->
+        <nav class="navbar navbar-expand-lg navbar-light bg-light">
+            <div class="container">
+                <a class="navbar-brand" href="#">
+                    <img src="<?php echo $GLOBALS['webroot']; ?>/sites/default/images/login_logo.gif" width="30" height="30" alt="">
+                    OpenEMR: Eye Exam <span class="font-weight-bold">Shorthand Help</span>
+                </a>
+                <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+                    <div class="navbar-nav">
+                        <a class="nav-item nav-link active" id="general_button" href="#">Introduction<span class="sr-only">(current)</span></a>
+                        <a class="nav-item nav-link" id="hpi_button" href="#">HPI</a>
+                        <a class="nav-item nav-link" id="pmh_button" href="#">PMH</a>
+                        <a class="nav-item nav-link" id="external_button" href="#">External</a>
+                        <a class="nav-item nav-link" id="antseg_button" href="#">Anterior Segment</a>
+                        <a class="nav-item nav-link" id="retina_button" href="#">Retina</a>
+                        <a class="nav-item nav-link" id="neuro_button" href="#">Neuro</a>
+                    </div>
+                </div>
+            </div>
+        </nav>
+        <!-- Content Section -->
+        <div class="container" name="container_group" >
+            <!-- Introduction Section -->
+            <div id="introductionAccordion">
+                <!-- Introduction Card Section -->
+                <div class="card">
+                    <div class="card-header">
+                        <h5 class="mb-0">
+                            <button class="btn btn-link text-decoration-none" data-toggle="collapse" data-target="#introduction" aria-expanded="true" aria-controls="collapseOne">
+                                Introduction: Paper vs. EHR
+                            </button>
```

Comment:
Hi @tywrenn 

Should I adjust the aria-controls value? or remove it?

Reply:
@stu01509 if it breaks upon removing it just leave it there otherwise you can remove it

Reply:
Ok, I will remove it.